docs/features/service_history.md

```diff
@@ -0,0 +1,190 @@
+# Service History
+
+The Service History subsystem provides a flattened, day-by-day representation of a client's journey through various programs. It is a primary source for some many reports and data analysis.
+
+## Key Models
+
+-   **`ServiceHistoryEnrollment`**: Each HUD enrollment produces one or more `ServiceHistoryEnrollment` rows that mark the lifecycle of the enrollment. We always create an `entry` record, we add an `exit` record when the enrollment closes, and we occasionally keep a `first` record for legacy compatibility. These markers allow downstream jobs to reason about the enrollment span independently of the daily service rows.
+-   **`ServiceHistoryService`**: This model stores the daily activity generated for a `ServiceHistoryEnrollment`. It contains both source services and synthetic days that are implied by enrollment rules (for example, extrapolated bed nights or contacts).
```

Comment:
The only extrapolation/synthetic records are Street Outreach, where configuration can be set to give someone the entire month of days for any street outreach current living situation collected.

Might be worth calling out that this table is partitioned by year to facilitate time-based reporting.